projects/agent-framework-javascript/agent-framework-javascript.md

```diff
@@ -0,0 +1,97 @@
+
+# TAC Sponsor
+
+Tracy Kuhrt
+
+# Project License
+
+The project has an Apache 2.0 license: [Aries Framework JavaScript - License)](https://github.com/hyperledger/aries-framework-javascript%23license)
+
+# Source Control
+
+The project is hosted in GitHub and includes the following repositories:
+
+- The framework [Aries Framework JavaScript)](https://github.com/hyperledger/aries-framework-javascript%23license)
+- The framework's extensions: [Aries Framework JavaScript Extensions](https://github.com/hyperledger/aries-framework-javascript-ext%23license)
+
+# Issue Tracker
+
+The issues are tracked using GitHub's Issues feature in the corresponding repository. For issues that span across multiple repositories, GitHub's Projects feature may be used.
+
+# External Dependencies
+
+Please see [Aries Framework JavaScript - Dependency Licenses](./dependency-licensing-overview.md)
+
+# Release Methodology
+
+All proposed repositories have continuous deployment/delivery pipelines built using [GitHub Actions](https://github.com/features/actions). The individual packages follow the [_semantic versioning_](https://semver.org/) method, ensuring consistency and safety. Whenever a pull request is merged into the `main` branch that doesn't cause a _major_, _minor_ or _patch_ version increase, an _alpha_ version is released.
+
+# Initial Maintainers
+
+| name               | Github                                                     |
+| ------------------ | ---------------------------------------------------------- |
+| Berend Sliedrecht  | [@blu3beri](https://github.com/blu3beri)                   |
+| Jakub Kočí         | [@jakubkoci](https://github.com/jakubkoci)                 |
+| James Ebert        | [@JamesKEbert](https://github.com/JamesKEbert)             |
+| Karim Stekelenburg | [@karimStekelenburg](https://github.com/karimStekelenburg) |
+| Timo Glastra       | [@TimoGlastra](https://github.com/TimoGlastra)             |
+| Ariel Gentile      | [@genaris](https://github.com/genaris)                     |
+| Jan Rietveld       | [@janrtvld](https://github.com/janrtvld)                   |
+
+# Proposed Project Governance
+
+The current governance model under Hyperledger is community-based. This means that decisions are made democratically, with the aim of community consensus. In cases where no clear consensus is established, active contributors may be granted a louder voice.
+
+This approach has proven effective and will be continued as the chosen governance model.
+
+# Links to Documented Governance Practices
+
+> Work in progress.
+
+# Financial Sponsorship
+
+Hyperledger has covered infrastructure related costs. Besides that, None.
```

Comment:
Will Hyperledger keep covering these costs after the move? If not, are the project's needs covered by what OWF provides? 

* https://tac.openwallet.foundation/governance/project-and-lab-services/
* https://tac.openwallet.foundation/governance/paid-tooling-policy/

Reply:
@skounis, these items are mentioned in the last section of the proposal. Do you think they should be listed here as well?